src/state.rs

```diff
@@ -18,140 +22,208 @@ pub struct OPRFServer {
     pub next_epoch_time: Option<String>,
 }
 
-/// Shareable wrapper around the server state
-pub type OPRFState = Arc<RwLock<OPRFServer>>;
-
-impl OPRFServer {
+impl OPRFInstance {
     /// Initialize a new OPRFServer state with the given configuration
     pub fn new(config: &Config) -> Result<Self, ppoprf::PPRFError> {
         // ppoprf wants a vector, so generate one from our range.
-        let epochs: Vec<u8> =
-            (config.first_epoch..=config.last_epoch).collect();
+        let epochs: Vec<u8> = (config.first_epoch..=config.last_epoch).collect();
         let epoch = epochs[0];
```

Comment:
This field should be removed in favour of the instance-specific current epochs.

Reply:
As in, we should remove first and last epoch switches?

Reply:
We automatically advance to the current epoch in `epoch_loop`

Reply:
Indeed. Sorry for the confusion.